code/__DEFINES/misc.dm

```diff
@@ -117,6 +117,13 @@
 	locate(min(CENTER.x+(RADIUS),world.maxx), min(CENTER.y+(RADIUS),world.maxy), CENTER.z) \
   )
 
+#define RANGE_EDGE_TURFS(RADIUS, CENTER)\
```

Comment:
Could you leave a comment on what this does? It's very hard to read right now.

Reply:
can do

Reply:
Done